service/consts_test.go

```diff
@@ -80,3 +80,37 @@ func deleteInsertGetSecret() (*pblib.Secret, error) {
 
 	return getActiveSecretRow()
 }
+
+func insertNewToken() (*pblib.Secret, string, error) {
+	// delete tokens table
+	_, err := postgresDB.Exec("DELETE FROM user_security.tokens")
+	if err != nil {
+		return nil, "", err
+	}
+
+	// delete secrets table and generate a new secret
+	newSecret, err := deleteInsertGetSecret()
+	if err != nil {
+		return nil, "", err
+	}
+	time.Sleep(2 * time.Second)
```

Comment:
I figure that this should be a good case for channels if we have to do something in a non-unit test source code.
https://hackernoon.com/using-channels-as-futures-in-golang-e085c4adbc04